### PR TITLE
Create alluxio.rst

### PR DESCRIPTION
**Pull request by dborkar**
Add new connector for Alluxio Data Orchestrator.

### CODE REVIEW

File: presto-docs/src/main/sphinx/connector/alluxio.rst [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+`Alluxio <www.alluxio.io>`_ is a data orchestration  open source distributed virtual file system which provides a block-level read/write caching engine for Presto connecting to a variety of disparate storage systems including S3 and HDFS. Presto can query files stored in Alluxio through the Hive connector. 
+Basic Configuration
+
+Presto can use the Hive Metastore for the database and table metadata information, as well as the file system location of table data. In order to enable this, create and edit the Presto configuration ``etc/catalog/hive.properties``:
+
+``connector.name=hive-hadoop2``
+``hive.metastore.uri=thrift://localhost:9083``
+
+Copy Alluxio client jar ``/<ALLUXIO_HOME>/client/alluxio-<VERSION>-client.jar`` into directory ``plugin/hive-hadoop2/`` on all Presto servers.
+Additional Alluxio Configuration
```
Comment (mattsfuller) [inline presto-docs/src/main/sphinx/connector/alluxio.rst:26]: Would it make sense to add a link to the Alluxio configuration options?

**Review by mattsfuller — APPROVED** [review]
This looks great. I made one minor suggestion about linking the the docs to the Alluxio properties.

File: presto-docs/src/main/sphinx/connector/alluxio.rst (hunk 1 above)
Comment (dborkar) [reply presto-docs/src/main/sphinx/connector/alluxio.rst:26]: Thanks for the feedback. Made it more modular and added some links.

**Comment by mosabua** [comment]
Can we format the source to 80 char lines please .. then I can review after

**Pull request closed without merging**